crates/bevy_core_pipeline/src/dof/mod.rs

```diff
@@ -110,7 +113,7 @@ pub struct DepthOfFieldSettings {
 }
 
 /// Controls the appearance of the effect.
-#[derive(Component, Clone, Copy, Default, PartialEq, Debug)]
+#[derive(Component, Clone, Copy, Default, PartialEq, Debug, Reflect)]
```

Comment:
We should reflect `PartialEq` here :)

Reply:
This is on the `DepthOfFieldMode` enum. Bloom has a similar enum where I also looked to see whats reflected. There, it's nothing, so I didn't reflect default.
So when I reflect `PartialEq`, should I also reflect `Default`?

Reply:
I believe so yes. @MrGVSV or @soqb should be able to advise more. My rule of thumb is "if a trait impl can be registered with reflection it should be".

Reply:
generally yes but `Default` is currently an awkward special case i've long since wanted to [resolve the semantics of](https://github.com/bevyengine/bevy/pull/9575). we encountered breakages when i tried.

notably, registering `#[reflect(Default)]` changes how `FromReflect` behaves when fields are not present. without `#[reflect(Default)]` all fields are always required (except those with `#[reflect(ignore)]` or `#[reflect(default)]`), whereas without it, no fields are required.